test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -83,15 +94,20 @@ public static IEnumerable<object[]> ValidateSetsMemberNamePropertyDataSet
         public void ValidateWithIsValidTrue()
```

Comment:
Could you give this a meaningful name while you're modifying this type?

`ValidateReturnsEmptyValidationResultSequence_IfIsValidReturnsTrue`?